### PR TITLE
Fix Interface Opening Exceptions

### LINKED ISSUE #28050: Open Interfaces unexpected exception
**Original reporter:** Found by @Harrietbrown 

### Expected behavior
Should be able to open each interface
### Steps to reproduce the behavior
Open Plot or Workbench, open any of the following interfaces:
- [x] Diffraction>HFIR 4Circle
Following error trace:
```
Traceback (most recent call last):
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 417, in <lambda>
    action.triggered.connect(lambda checked_py, script=script: self.launch_custom_python_gui(script))
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 377, in launch_custom_python_gui
    self.interface_executor.execute(open(filename).read(), filename)
  File "C:\MantidNightlyInstall\bin\mantidqt\widgets\codeeditor\execution.py", line 155, in execute
    exec (code_obj, self.globals_ns, self.globals_ns)
  File "C:/MantidNightlyInstall/scripts/HFIR_4Circle_Reduction.py", line 12, in <module>
    from HFIR_4Circle_Reduction import reduce4circleGUI  # noqa
  File "C:\MantidNightlyInstall\scripts\HFIR_4Circle_Reduction\reduce4circleGUI.py", line 34, in <module>
    from HFIR_4Circle_Reduction import PreprocessWindow
  File "C:\MantidNightlyInstall\scripts\HFIR_4Circle_Reduction\PreprocessWindow.py", line 10, in <module>
    import reduce4circleControl
ModuleNotFoundError: No module named 'reduce4circleControl'
```
- [x] Direct>DGSPlanner

```
Traceback (most recent call last):
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 417, in <lambda>
    action.triggered.connect(lambda checked_py, script=script: self.launch_custom_python_gui(script))
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 377, in launch_custom_python_gui
    self.interface_executor.execute(open(filename).read(), filename)
  File "C:\MantidNightlyInstall\bin\mantidqt\widgets\codeeditor\execution.py", line 155, in execute
    exec (code_obj, self.globals_ns, self.globals_ns)
  File "C:/MantidNightlyInstall/scripts/DGSPlanner.py", line 14, in <module>
    planner = DGSPlannerGUI.DGSPlannerGUI()
  File "C:\MantidNightlyInstall\scripts\DGSPlanner\DGSPlannerGUI.py", line 55, in __init__
    self.instrumentWidget = InstrumentSetupWidget.InstrumentSetupWidget(self)
  File "C:\MantidNightlyInstall\scripts\DGSPlanner\InstrumentSetupWidget.py", line 220, in __init__
    self.updateFigure()
  File "C:\MantidNightlyInstall\scripts\DGSPlanner\InstrumentSetupWidget.py", line 274, in updateFigure
    self.gonfig.set_aspect('equal', adjustable='datalim')
  File "C:\MantidNightlyInstall\bin\lib\site-packages\matplotlib\axes\_base.py", line 1280, in set_aspect
    raise NotImplementedError(
NotImplementedError: It is not currently possible to manually set the aspect on 3D axes
```
- [x] Utility>QECoverage
```
Traceback (most recent call last):
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 417, in <lambda>
    action.triggered.connect(lambda checked_py, script=script: self.launch_custom_python_gui(script))
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 377, in launch_custom_python_gui
    self.interface_executor.execute(open(filename).read(), filename)
  File "C:\MantidNightlyInstall\bin\mantidqt\widgets\codeeditor\execution.py", line 153, in execute
    code_obj = compile(block.code_str, filename, mode=COMPILE_MODE,
  File "C:/MantidNightlyInstall/scripts/QECoverage.py", line 461
    def indirect_input_check(self, ana):
    ^
IndentationError: unexpected indent
```
(might this last one be fixed by #28048 ?)
QEcoverage DOES open in MantidPlot...
and it opens in a Workbench build of current master
### Platforms affected
Tested on 4.2.20200211.2150 Nightly Workbench and Plot

DGSPlanner issue does not affect Ubuntu.

### PR DESCRIPTION
**Pull request by cailafinn**
Testing for this PR should be done on *at least* windows and one other OS before being approved. 

**Description of work.**
 - Fixed imports in HFIR Diffraction GUI
- Removed `set_aspect` call on 3d plot which never worked properly on 3d plots and was removed from matplotlib 3.1 and was stopping the Direct DGS planner opening. 
- QECoverage didn't seem to require changes and may have been fixed by a previous PR. 
 
**To test:**
1. Check all interfaces open, particularly:
a. Diffraction -> HFIR 4Circle
b. Direct -> DGSPlanner
c. Utility -> QECoverage

Fixes #28050 

*This does not require release notes* because **it fixes issues not present in the previous release.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/DGSPlanner/InstrumentSetupWidget.py [hunk 1]
```diff
@@ -271,7 +271,6 @@ def updateFigure(self):
         self.gonfig.text(1,0,-2.5,'X',zdir=None,color='black')
         self.gonfig.plot([0,0],[-3,-3],[-2,-0.5],zdir='y',color='black',linewidth=3)
         self.gonfig.text(0,-1,-2.5,'Beam',zdir=None,color='black')
-        self.gonfig.set_aspect('equal', adjustable='datalim')
```
Comment (DavidFair) [inline scripts/DGSPlanner/InstrumentSetupWidget.py]: Was this completely removed or did the api get renamed? I'm concerned we may unintentionally be changing the default behaviour here

Comment (martyngigg) [reply scripts/DGSPlanner/InstrumentSetupWidget.py]: It turns out this basically did completely the wrong thing in matplotlib: see matplotlib/matplotlib#13474 and it now throws a `NotImplementedError`. I guess we could catch it but it's not clear that it's actually ever going to be fixed.

Comment (ConorMFinn) [reply scripts/DGSPlanner/InstrumentSetupWidget.py]: Also, if it did the wrong thing anyway, do we really want it to continue doing that?

Comment (DavidFair) [reply scripts/DGSPlanner/InstrumentSetupWidget.py]: I'm happy with the removal, since it could change under us if upstream re-implement it

**Review by DavidFair — APPROVED** [review]
Went through and all the interfaces currently open for me. Lets :shipit:

**Pull request merged**